app/components/support_interface/ucas_match_action_component.rb

```diff
@@ -0,0 +1,60 @@
+module SupportInterface
+  class UCASMatchActionComponent < ViewComponent::Base
+    include ViewHelper
+
+    ACTIONS = {
+      send_initial_emails: {
+        description: 'Send initial emails',
+        button_text: 'Confirm initial emails were sent',
+        form_path: :support_interface_record_initial_emails_sent_path,
+      },
+    }.freeze
+
+    def initialize(match)
+      @match = match
+    end
+
+    def inset_text_header
+      return 'No action required' if !@match.dual_application_or_dual_acceptance? || !@match.action_needed?
```

Comment:
I wasn't sure about this. I could just do `unless @match.action_needed?` here if I changed the `action_needed?` to something like:

```
  def action_needed?
    return false if processed?

    return false unless dual_application_or_dual_acceptance?

    return true if ucas_withdrawal_requested?

    return need_to_send_reminder_emails? if initial_emails_sent?

    return need_to_request_withdrawal_from_ucas? if reminder_emails_sent?

    true
  end
```
We use `action_needed?` a lot on the index page, in the filters, for slack notifications, so with performance in mind I wanted to keep the `dual_application_or_dual_acceptance?` as the last check in this method since it goes through matching data of each UCAS match. 
But on that line we actually need to check `dual_application_or_dual_acceptance?` first.

Do you think I should change it as shown above or create some other method to capture `!@match.dual_application_or_dual_acceptance? || !@match.action_needed?`? 🤔

Reply:
Added a commit d2d8ab7